boards/arm/efm32pg_stk3402a/doc/index.rst

```diff
@@ -6,9 +6,9 @@ EFM32 Pearl Gecko Starter Kit
 Overview
 ********
 
-The EFM32 Pearl Gecko Starter Kit EFM32PG-STK3402A contains an MCU from the
-EFM32PG family built on an ARM® Cortex®-M4F processor with excellent low
-power capabilities.
+The EFM32 Pearl Gecko Starter Kit EFM32PG-STK3402A contains an MCU from
+the EFM32PG family built on an ARM® Cortex®-M4F processor with excellent
+low power capabilities.
```

Comment:
Please leave the original formatting. It's much easier to review your changes if you reduce the diff log to a minimum.

Reply:
If you cannot tell, I'm giving our developers some training in converting our Confluence documentation back into Zephyr, and I got it much closer that it appears, but I will go try again to see if I can minimize the changes even more.